Week-2/Excersice 2.1/LoremIpsum.html

```diff
@@ -0,0 +1,134 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
we can have some meaniful file name instead of LoremIpsum.html.. also PR description..